src/main/kotlin/uk/gov/dluhc/printapi/messaging/models/ProcessPrintBatchStatusUpdateMessage.kt

```diff
@@ -0,0 +1,6 @@
+package uk.gov.dluhc.printapi.messaging.models
+
+data class ProcessPrintBatchStatusUpdateMessage(
```

Comment:
Rather than define the model here, please can you define it in `print-api-sqs-messaging.yaml` and allow it to be auto generated (as per what Fayi has done in his PR)

Reply:
@NathanRussellValtech Ok note, will update spec and generate this in my next PR where listener is processing this message. So I will make these changes in next PR if that's ok?

Reply:
Yes, that's fine 😁 
PRs "little and often", iteratively build and improve the code; that's 100% fine with me 👍 😁

Reply:
@NathanRussellValtech tiny [PR](https://github.com/cabinetoffice/eip-ero-print-api/pull/36) raised for this change